### DISCUSSION TITLE
Is there a way to clear multiple collections?

### DISCUSSION THREAD

**Discussion by flyj222** [isar/isar#769] [Help] [answered]
Currently I'm developing an app that wants to wipe local data stored in Isar when the user logouts.
I have multiple isarCollections in this app.

The usual way to clear collections is like the following:
```
await isar.writeTxn(() async {  
  isar.<collection1>.clear();  
  isar.<collection2>.clear();  
  .....  
});
```

I wanted to know is there a way to programatically get the IsarCollections so I can streamline the code.
Is there a way to get the collection through the CollectionSchema object?
Any help is greatly appreciated!

**Comment by Jtplouffe** [comment, marked as answer]
If you want to clear every collections, you can use `isar.clear()`, which will loop through all collections and clear them.
